test/03_USM.test.js

```diff
@@ -255,186 +212,369 @@ contract('USM', (accounts) => {
             )
           })
 
-          it("decays mintBurnAdjustment over time", async () => {
-            // Check that mintBurnAdjustment decays properly (well, approximately) over time:
-            // - Our adjustment was previously 1/4: see mintBurnAdj2 above.
-            // - Use timeMachine to move forward 180 secs = 3 min.
-            // - Since USM.BUY_SELL_ADJUSTMENTS_HALF_LIFE = 60 (1 min), 3 min should mean a decay of ~1/8.
-            // - The way our "time decay towards 1" approximation works, decaying 1/4 by ~1/8 should yield 1 + (1/4 * 1/8) - 1/8 = 0.90625.
-            //   (See comment in USM.mintBurnAdjustment().)
-            // - And finally - since our decay's exponentiation is approximate (WadMath.wadHalfExp()), don't compare mintBurnAdj3 to
-            //   targetMintBurnAdj3 *exactly*, but instead to within some reasonable error (eg, 10).
-            const mintBurnAdj2 = (await usm.mintBurnAdjustment())
-            
-            const block0 = (await web3.eth.getBlockNumber())
+          it("uses flat USM price first time USM is minted", async () => {
```

Comment:
Does this test anything?

Reply:
Well, it tests that the first time `mint()` is called, the price per USM is just linear (`wadMul(ethPerMint, priceWAD)` a few lines down) rather than on a sliding scale, as it will be for all future calls.  This is a "make sure things are working as expected" test rather than a "make sure something bad doesn't happen" test.